packages/tracking/src/hooks/use_track.ts

```diff
@@ -25,16 +25,15 @@ declare const window: Window & {
 
 
 type TrackEventMap = {
-  'gav4_clickEvent': ClickEvent;
-  'gav4_pageView': PageViewEvent;
-  'gav4_removeEvent': RemoveEvent;
-  'gav4_editEvent': EditEvent;
-  'gav4_success_failEvent': SuccessFailEvent;
-  'gav4_reactEvent': ReactEvent;
-  'gav4_searchEvent': SearchEvent;
-  'gav4_swipeEvent': SwipeEvent;
-  'gav4_viewEvent': ViewEvent;
-  'gav4_unknownEvent': UnknownEvent;
+  'click': ClickEvent;
+  'page_view': PageViewEvent;
+  'remove': RemoveEvent;
+  'edit': EditEvent;
```

Comment:
not in the list of events in the JIRA ticket but probably used somewhere :-p

Reply:
it was defined in other wiki pages, did not implement every event specified in the current ticket yet